Sources/DequeModule/Deque._UnsafeHandle.swift

```diff
@@ -188,8 +188,8 @@ extension Deque._UnsafeHandle {
     assert(offsets.lowerBound >= 0 && offsets.upperBound <= count)
     let lower = slot(forOffset: offsets.lowerBound)
     let upper = slot(forOffset: offsets.upperBound)
-    if offsets.count == 0 || lower < upper {
```

Comment:
I don't expect `Range.count == 0` to produce less efficient code than `Range.isEmpty` -- do you have proof that it matters?

Reply:
No, I may be mistaken. Since it is a RandomAccessCollection type, I don’t think there will be any difference.